uploader/pantheon.py

```diff
@@ -108,6 +123,7 @@ def _print_response(filetype, path, response_code, reason):
 
 server: http://localhost:8080
 repository: pantheonSampleRepo
+attributeFile: path/to/attribute.adoc
```

Comment:
Is camel-casing the right standard for yaml files?
Also, are you envisioning this as an attributes' file for the whole repository? or just modules? If the latter, I would consider naming this something like `moduleAttributesFile`

Reply:
camelCase is the right standard for everything. :)

Reply:
kubernetes use camelCase:
https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/

ansible uses snake_case:
https://github.com/ansible/ansible-examples/blob/master/lamp_simple/site.yml

Reply:
@carlosmunoz i'm envisioning it as an attribute fiel for the whole repository.